src/components/pages/TopPage/hooks.ts

```diff
@@ -24,7 +24,7 @@ export default () => {
   }, [isAuthenticated, navigate, currentTeam, setLocalState, data, teamId]);
 
   useEffect(() => {
-    if (authError || (!loading && !data?.me)) {
+    if (authError || (!loading && data?.me === null)) {
```

Comment:
I am not sure if this will call before or after the query starts so is it better (like @rot1024 suggested) to make it `(isAuthenticated &&  !loading && data?.me === null)`.

Reply:
Thanks! I fixed it.